pom.xml

```diff
@@ -27,9 +27,10 @@
 
         <!-- test -->
         <version.wildfly>8.2.1.Final</version.wildfly>
-        <version.cdi-api>1.2</version.cdi-api>
-        <version.ejb3>1.0.0.Final</version.ejb3>
+        <version.cdi-api>2.0.Alpha5</version.cdi-api>
```

Comment:
Is it really necessary? CDI 2.0 is not final yet and we don't really use any of the API addons offered by this tool. Moreover target containers we run our test against are still on CDI 1.x. I think we should revert.